### PR TITLE
Update stars demo to use latest policy agent

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: docs/cni/kubernetes/VagrantCoreOS.md [hunk 1]
```diff
@@ -50,16 +50,20 @@ To connect to your servers
...
-You should now have two CoreOS servers - one Kubernetes master and one Kubernetes node. The servers are named calico-01 and calico-02
+You should now have three CoreOS servers - one Kubernetes master and two Kubernetes nodes. The servers are named k8s-master, k8s-node-01, and k8s-node-02 
 and have IP addresses 172.18.18.101 and 172.18.18.102.
 
 At this point, it's worth checking that your servers can ping each other.
 
-From calico-01
+From k8s-master 
 
     ping 172.18.18.102
```
Comment (robbrockbank) [inline docs/cni/kubernetes/VagrantCoreOS.md]: and `ping 172.18.18.103`

File: docs/cni/kubernetes/VagrantCoreOS.md [hunk 2]
```diff
@@ -50,16 +50,20 @@ To connect to your servers
 
 ### 1.4 Verify environment
 
-You should now have two CoreOS servers - one Kubernetes master and one Kubernetes node. The servers are named calico-01 and calico-02
+You should now have three CoreOS servers - one Kubernetes master and two Kubernetes nodes. The servers are named k8s-master, k8s-node-01, and k8s-node-02 
 and have IP addresses 172.18.18.101 and 172.18.18.102.
```
Comment (robbrockbank) [inline docs/cni/kubernetes/VagrantCoreOS.md]: 172.18.18.101, 172.18.18.102 and 172.18.18.102

**Comment by caseydavenport** [comment]
@robbrockbank Can I merge this?

**Comment by robbrockbank** [comment]
@caseydavenport Ah my bad!  I'd meant to say LGTM with a couple of minor comments....  Merging now.

**Pull request merged**